widgetssdk/src/main/java/com/glia/widgets/call/CallController.kt

```diff
@@ -129,9 +131,17 @@ internal class CallController(
     // This is done to prevent errors such as "does not support one-way audio calls" only because the
     // other participant media state is passed few milliseconds late.
     private fun subscribeToMediaState() {
+        val dummyState = object : MediaState {
```

Comment:
It can be renamed to initialState or initialMediaState, I suppose.

Reply:
fixed d61ebfc1b85167b43c48705c7bb76b994037a50f